sdk/js/src/CacheStore.ts

```diff
@@ -60,6 +81,31 @@ export abstract class CacheStore {
         const anonUserId = this.load(StoreKey.AnonUserId)
         return anonUserId ? JSON.parse(anonUserId) : anonUserId
     }
+
+    saveConfigUserId(user: DVCPopulatedUser): void {
+        const { user_id: userId, isAnonymous } = user
+        const userIdKey = this.getUserIdStoreKey(user)
+        this.save(userIdKey, userId)
+        this.logger?.info(`Successfully saved ${isAnonymous ? 'anonymous': 'identified'} user id to local storage`)
+    }
+
+    loadConfigUserId(user: DVCPopulatedUser): string | null | undefined {
+        const userIdKey = this.getUserIdStoreKey(user)
+        const userId = this.load(userIdKey)
+        return userId ? JSON.parse(userId) : userId
+    }
+
+    saveConfigFetchDate(user: DVCPopulatedUser): void {
```

Comment:
the user ID and fetch date associated with the config should only be updated when the config is updated, so we shouldn't have methods for setting them individually